### PR TITLE
dnscontrol: update livecheck

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

3.10.5 did get recalled in the past, and now the new tag have not been tagged as `latest release` for quite some time, it is safe to assume the `latest release` would be a better route to maintain the formula upgrade.

### CODE REVIEW

File: Formula/dnscontrol.rb
Comment (samford) [inline Formula/dnscontrol.rb:11]:
```suggestion
  # released and they sometimes re-tag versions before that point, so it's
  # necessary to check release versions instead of tags.
```

I was initially unsure what "recall" entailed here but it sounds like it's just re-tagging a version. Explaining it that way may be clearer to other maintainers, so I've tweaked the language here.

**Review by samford — COMMENTED** [review]
I'm always forgetting that I can't approve after pushing the last commit but this is good to merge after someone approves it (and we fix CI).

**Review by carlocab — APPROVED** [review]

File: Formula/dnscontrol.rb
Comment (chenrui333) [reply Formula/dnscontrol.rb]: yeah, from the tag reuse perspective, it could be considered a re-tagging, but the upstream kind of abused it by `recall and re-tag` IMO 😅

**Pull request merged**
